transformation/src/main/resources/logback.xml

```diff
@@ -5,11 +5,22 @@
         </encoder>
     </appender>
 
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <file>../logs/errors.log</file>
+        <encoder>
+            <pattern>%msg%n</pattern>
+        </encoder>
+    </appender>
+
     <!-- Use debug level for our own messages so we can eyeball logs during testing. -->
     <logger name="org.broadinstitute" level="DEBUG" additivity="false">
         <appender-ref ref="STDOUT" />
     </logger>
 
+    <logger name="org.broadinstitute" level="ERROR" additivity="false">
+        <appender-ref ref="FILE" />
+    </logger>
+
```

Comment:
These changes just make the logger write any error messages to a file. They don't change the way our other logging works.